### ISSUE TITLE
accuracy mismatch and logger is empty

### ISSUE THREAD

**Issue by Urheen** [NUS-HPC-AI-Lab/Neural-Network-Diffusion#15]
Hi authors:

after running cifar100_resnet18_k200.sh, I found that the results is 76.09%, but what reported in the paper is higher than it.

![image](https://github.com/NUS-HPC-AI-Lab/Neural-Network-Diffusion/assets/27939296/bbe16319-900d-4102-9d17-e08145267b5e)

Furthermore, I found that the logger file "./outputs/cifar100/train_p_diff.log" is empy.
Could you please help with this issue?

Best

**Comment by 1zeryu** [comment]
The performance of generated models is related to your training data that is training using SGD. 
Would you provide more details like original model performance? The SGD-trained model performance may be affected by many factors, i.e., random seed, GPU type, so the 76.09% maybe reasonable results.
The train_p_diff.log contains terminal output about experiment initialization and torch.distributed.nn. This does not affect the conduct of the experiment.
If still have any questions or concerns, feel free to contact us.

**Comment by Urheen** [comment]
Hi,

You mentioned, "The train_p_diff.log contains terminal output about experiment initialization and torch.distributed.nn. This does not affect the conduct of the experiment."

However, after running your code, I found that the log file is empty. Everything is shown in the terminal but is not saved to the local log file.
